lib/ddtrace/contrib/sidekiq/server_internal_tracer/job_fetch.rb

```diff
@@ -0,0 +1,23 @@
+# typed: true
+
+module Datadog
+  module Contrib
+    module Sidekiq
+      module ServerInternalTracer
+        # Trace job fetches
+        module JobFetch
+          private
+
+          def fetch
+            Datadog.configuration[:sidekiq][:tracer].trace(Ext::SPAN_JOB_FETCH) do |span|
+              span.service = Datadog.configuration[:sidekiq][:service_name]
```

Comment:
Because access to `Datadog.configuration[:sidekiq]` is not as cheap as a hash access, I recommend you access it only once when possible.
```suggestion
            configuration = Datadog.configuration[:sidekiq]
            configuration[:tracer].trace(Ext::SPAN_JOB_FETCH) do |span|
              span.service = configuration[:service_name]
```

Reply:
Good to know, updated!